vllm/model_executor/layers/rejection_sampler.py

```diff
@@ -59,6 +60,10 @@ def __init__(self,
         else:
             logger.info("Use pytorch for rejection sampling.")
 
+        if os.environ.get('VLLM_MTP_PRINT_ACCPET_RATE', '0') != '0':
```

Comment:
How about change it to:
os.environ.get('VLLM_MTP_PRINT_ACCPET_RATE', '0').lower() in ('true', '1'):